Automattic/newspack-popups
# Node v12 Support

### All Submissions:






### Changes proposed in this Pull Request:

Updates Calypso Build so the plugin can be built with current Node.

### How to test the changes in this Pull Request:

1. `nvm use  stable`,  then check `node --version`
2. `npm install`
3. `npm start`
4. Verify plugin builds successfully with no errors.

### Other information:
